src/routes/buildManifestObject.js

```diff
@@ -22,6 +23,8 @@ const getQueryTemplateValue = (queryValue, param) => {
     : `${param}>="${queryValue}" or ${param}<"${queryValue}"`;
 };
 
+const getQueryTemplateSubjects = (queryValue) => `subjects==/string "${queryValue}"`;
```

Comment:
Should it be `==\` as in [the docs](https://github.com/folio-org/mod-search#cql-query-modifiers)?

Reply:
The ticket said :
update query parameters from query =(subjects="subject") to query =(**subjects ==/string "subject")** when the user clicks on the row with a given subject so that the list of accurate list of matching records is returned.